### DISCUSSION TITLE
Höhendiagramm

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#453] [Q&A] [answered]
Hello
Ist es möglich das Höhendiagramm während der Planung auszublenden?
Lg.

**Comment by 1maxi2** [comment]
Sorry hab es gerade im Menü Routing gefunden. Beitrag kann gelöscht werden

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

The elevation diagram in route planning can be switched on / off in the routing settings and in the route menu.

Das Höhendiagramm in der Routenplanung kann in den Routingeinstellungen und im Routenmenü ein- und ausgeschaltet werden.
